client/elements/sc-page-selector.html

```diff
@@ -345,20 +350,33 @@
               }
           }
 
+          _createMetaData(pageTitle) {
+              const today = new Date();
+              this.metaData = {
+                'og:title': pageTitle,
+                'og:description': "Early Buddhist texts and modern translations. Suttas (sutras) from the Tipitaka (Tripitaka) in Pali, Chinese, Sanskrit, and Tibetan with the Buddha's teachings on mindfulness, insight, wisdom, and meditation.",
+                cdate: today,
+                mdate: today
+              };
+          }
+
           _getPageTitle(title) {
+              let pageTitle = '';
               if (title === null) {
-                  return 'Page not found';
+                  pageTitle = 'Page not found';
               }
               else if (['searchResults', 'dictionaryResults'].includes(title)) {
-                  return this.localize(title);
+                  pageTitle = this.localize(title);
               }
               else if (this.suttaId && this.shouldShowSuttaTextPage && title) {
-                  return `${this.suttaId.toUpperCase()}: ${title}`;
+                  pageTitle = `${this.suttaId.toUpperCase()}: ${title}`;
               } else if (title) {
-                  return title;
+                  pageTitle = title;
               } else {
-                  return 'SuttaCentral';
+                  pageTitle = 'SuttaCentral';
               }
+              this._createMetaData(pageTitle);
```

Comment:
^ explanation from above: With this line, this function now has side effects, whereas it should be a simple getter function.